Resources/Prototypes/Loadouts/Jobs/Medical/traitRestrictedGear.yml

```
@@ -0,0 +1,445 @@
+  category: JobsMedicalMedicalDoctor
+  cost: 4
+  requirements:
+    - !type:CharacterTraitRequirement
+      traits:
+        - RestrictedGear
+    - !type:CharacterJobRequirement
+      inverted: true
+      jobs:
+        - MedicalDoctor
+  items:
+    - ClothingBeltMedicalFilled
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)